src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -427,7 +427,7 @@ export const enum OpenEditorContext {
 	COPY_EDITOR = 3
 }
 
-export interface IEditorGroup {
+export interface IReadonlyEditorGroup {
```

Comment:
Given my suggestion to work on the model directly, we would not need this distinction anymore because this here refers to the viewer and not the model.